Source/Scene/Model.js

```diff
@@ -5317,7 +5328,12 @@ Model.prototype.update = function (frameState) {
         loadResources.resourcesParsed &&
         loadResources.pendingShaderLoads === 0
       ) {
-        ModelOutlineLoader.outlinePrimitives(this);
+        console.log("Back Face Culling: " + this.backFaceCulling); //remove after testing in sandcastle
+        console.log("Show: " + this.show); //remove after testing in sandcastle
+        console.log("Ignore Outline: " + this.ignoreOutline); //remove after testing in sandcastle
```

Comment:
I had to add the `ignoreOutline` option to `Batched3DModel3DTileContent.initialize` where it calls the Model constructor. In your Sandcastle, the debugger call stack showed that `Model` was being called from there so I took a look at the function. There are other places where a Model is created with a tileset's options, i.e. `Gltf3DTileContent` - maybe if this is only a change we want to apply to OSM Buildings then we don't need to update the option anywhere else?

Reply:
I think that this change should only apply to OSM Buildings. I just updated `Gltf3DTileContent` accordingly. However, I am open to feedback from our team and the community on how broad the ignore outlines functionality should be.